addons/mail/static/src/core/common/message.js

```diff
@@ -526,6 +526,10 @@ export class Message extends Component {
         this.state.showTranslation =
             !this.state.showTranslation && Boolean(message.translationValue);
     }
+
+    get removeFromThread() {
```

Comment:
The name feels a bit too generic and it looks like the name of a function and not of a boolean, even though it is just used in a very specific flow.

Maybe something like `shouldHideFromMessageListOnDelete`?